knex_migrations/20190612141100_allowLongerText.js

```diff
@@ -0,0 +1,18 @@
+exports.up = async function(knex, Promise) {
+  const { client } = knex.client.config;
+
+  if (client === "mysql") {
+    await knex.schema.alterTable("content_revisions", table => {
+      table.text("data", "mediumtext").alter();
+    });
+  }
+};
+
+exports.down = async function(knex, Promise) {
+  const { client } = knex.client.config;
+  if (client === "mysql") {
+    await knex.schema.alterTable("content_revisions", table => {
+      table.text("data").alter();
```

Comment:
Sure that this is the correct backwards migration?

Reply:
If you don't specify a textType, it uses the default "text".